test_wazuh/test_fim/test_skip/data/configure_nfs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+rpm -qa | grep -E "nfs-utils.*" || yum -y install nfs-utils
```

Comment:
This script should work in Centos, Debian and FreeBSD.

Reply:
Done